src/WebJobs.Extensions.DurableTask/HttpApiHandler.cs

```diff
@@ -1146,8 +1215,12 @@ internal HttpCreationPayload GetInstanceCreationLinks()
             string sendEventPostUri,
             string terminatePostUri,
             string purgeHistoryDeleteUri,
-            string restartPostUri)
+            string restartPostUri,
+            string suspendPostUri,
+            string resumePostUri)
         {
+            string instancePrefix = this.GetBaseUrl() + "/" + InstancesControllerSegment + WebUtility.UrlEncode(instanceId);

```

Comment:
Looks like this local bariable is not being used?

Reply:
I have removed it.